src/pages/ProblemEditor/Dialogs/FailureDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from "@mui/material";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { useAppSelector } from "../../../app/common/hooks";
+import { closeFailureModal } from "../ProblemEditorContainer/problemEditorContainerSlice";
+
+interface Props {}
+
+export const FailureDialog = (props: Props) => {
```

Comment:
I think you might as well not have props but ignore this if you might add some in the future or if its for stylistic reasons.